frontend/src/metabase/admin/settings/components/widgets/AdminSettingInput.tsx

```diff
@@ -281,15 +281,15 @@ export function SetByEnvVarWrapper<SettingName extends SettingKey>({
 }: SetByEnvVarWrapperProps<SettingName>) {
   if (
     settingDetails &&
-    settingDetails.is_env_setting &&
-    settingDetails.env_name
+    settingDetails?.is_env_setting &&
```

Comment:
Same with these additional `?.`. The if condition makes it so that it's not necessary